app/main.py

```diff
@@ -1 +1,31 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.health = health
+        self.name = name
+        self.hidden = False
+        Animal.alive.append(self)
```

Comment:
Will it works if I want to create an instance with the `hidden = True` parameter?

Reply:
> Will it works if I want to create an instance with the `hidden = True` parameter?

Sure not, but I wasn't sure either we going to inherit it and change only for rare classes, how it should be in ideal case(Which animal takes hidden by default?) or it should be hidden, and since it passed tests, I was sure I made it right way, as said in README.MD -> according to nature's logic. 
sry )